support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -31,6 +31,10 @@ Mappings:
       RedemptionTables:
       - arn:aws:dynamodb:*:*:table/redemption-codes-DEV
       - arn:aws:dynamodb:*:*:table/redemption-codes-UAT
+      S3Files:
```

Comment:
support-workers now needs to be able to load the product catalog from S3 to be able to work out the price of a supporter plus sub and from that how much of the purchase price is actually a contribution.